website/content/en/docs/deployment/_index.md

```diff
@@ -3,3 +3,12 @@ title = "Deployment"
+
+If you want to deploy Kubeflow with minimal changes, but optimized for [Amazon Elastic Kubernetes Service](https://aws.amazon.com/eks/) (Amazon EKS), then consider the [vanilla](/kubeflow-manifests/docs/deployment/vanilla/guide/) deployment option. The Kubeflow on AWS control plane is installed on top of Amazon EKS, which is a managed container service used to run and scale Kubernetes applications in the cloud.
+
+Kubeflow on AWS provides integration with the [Amazon Relational Database Service](https://aws.amazon.com/rds/) (RDS) for highly scalable and available pipelines and metadata store. RDS removes the need to manage a local MYSQL database service and storage. Kubeflow on AWS also provides integration with [Amazon Simple Storage Service](https://aws.amazon.com/s3/) (S3) for an easy-to-use pipeline artifacts store. S3 removes the need to host the local object storage MinIO. You can deploy Kubeflow on AWS with both [RDS and S3](/kubeflow-manifests/docs/deployment/rds-s3/guide/) or [using only RDS or only S3](/kubeflow-manifests/docs/deployment/rds-s3/guide/#using-only-rds-or-only-s3).
+
+Integrate your deployment with [AWS Cognito](https://aws.amazon.com/cognito/) for user authentication, which removes the complexity of managing user pools or connectors in Kubeflow’s native Dex authorization service. For more information, see the [Cognito deployment guides](/kubeflow-manifests/docs/deployment/cognito/). You can also deploy Kubeflow on AWS with all available service integrations by following the [Cognito, RDS, and S3 deployment guide](/kubeflow-manifests/docs/deployment/cognito-rds-s3/guide/). 
+
+Experiment with Kubeflow on AWS add-ons for additional service integrations. For Kubeflow notebook Workspace Volumes and Data Volumes, see the [Amazon Elastic File System](https://aws.amazon.com/efs/) (EFS) [add-on guide](/kubeflow-manifests/docs/deployment/add-ons/storage/efs/guide/) or the [Amazon FSx for Lustre](https://aws.amazon.com/fsx/lustre/) (Amazon FSx) [add-on guide](/kubeflow-manifests/docs/deployment/add-ons/storage/fsx-for-lustre/guide/) for increased training performance. You can integrate with [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/) for persistent log management, which addresses the default K8s log limits and improves your log availability and monitoring capabilities. For more information see the CloudWatch [add-on guide](/kubeflow-manifests/docs/deployment/add-ons/cloudwatch/guide/). You can also add secure external traffic management to the Kubeflow dashboard using the [AWS Application Load Balancer](https://aws.amazon.com/elasticloadbalancing/application-load-balancer/) (ALB) [add-on guide](/kubeflow-manifests/docs/deployment/add-ons/load-balancer/guide/).
```

Comment:
Experiment with Kubeflow on AWS add-ons for additional service integrations.

experiment? Lets start with something along the lines of. Distro offers additional service integrations which can be used with any Kubeflow deployment option. For e.g.,